src/test/java/com/exasol/adapter/document/ExasolTestDatabaseBuilder.java

```diff
@@ -87,24 +87,27 @@ public void dropConnection(final String name) throws SQLException {
         this.getStatement().execute("DROP CONNECTION IF EXISTS " + name);
     }
 
-    protected void addDebuggerOptions(final StringBuilder statementBuilder, final boolean isUdf) {
+    protected String getDebuggerOptions(final boolean isUdf) {
         final String hostIp = this.testInterface.getTestHostIpAddress();
-        final StringBuilder jvmOptions = new StringBuilder();
+        final StringBuilder jvmOptionsBuilder = new StringBuilder();
         if (hostIp != null) {
-            jvmOptions.append("-javaagent:/buckets/bfsdefault/default/").append(JACOCO_JAR_NAME)
-                    .append("=output=tcpclient,address=").append(hostIp).append(",port=3002");
+            jvmOptionsBuilder.append("-javaagent:/buckets/bfsdefault/default/" + JACOCO_JAR_NAME
```

Comment:
I think in this case append looks also readable, but you can choose what you like more because it's still test